packages/quantification/src/__tests__/quantification.test.ts

```diff
@@ -166,7 +167,7 @@ describe('getQuantificationSummary', () => {
           '2021': 92.358_872_428_480_85,
           '2022': 57.860_299_141_229_15,
           '2023': 75.209_361_959_482_37,
-          '2024': Number.NaN,
+          '2024': 49.259_094_477_479_096,
```

Comment:
Why did these test results change? Is this because of the changes to example-output.ts?

Reply:
Yeah. Some of these test fixtures are almost 2 years old, and don't reflect the current requirements of our data. So I had to update them with values.